bin/import-borders

```diff
@@ -4,10 +4,10 @@ set -o pipefail
 set -o nounset
 
 : "${IMPORT_DIR:=/import}"
-# Do not use .pbf extension to avoid accidental conflict with the original pbf with multiple runs
+: "${BORDERS_CLEANUP:=false}"
```

Comment:
I suggest the cleanup should be turned on by default.
I believe most users do not use Openmaptiles for large extracts such as planet. Moreover, those who do use it for planet are usually more skilled and would know the internals better.

Reply:
ok, agree to both changes - could you submit them and i will merge?  Assuming you are ok with the other changes i made?

Reply:
Done.